code/modules/antagonists/heretic/magic/caretaker.dm

```diff
@@ -24,12 +24,16 @@
 /datum/action/cooldown/spell/caretaker/is_valid_target(atom/cast_on)
 	return isliving(cast_on)
 
-/datum/action/cooldown/spell/caretaker/cast(atom/cast_on)
+/datum/action/cooldown/spell/caretaker/cast(mob/living/cast_on)
 	. = ..()
 	for(var/mob/living/alive in orange(5, owner))
 		if(alive.stat != DEAD && alive.client)
 			owner.balloon_alert(owner, "other minds nearby!")
 			return FALSE
+	
+	if(locate(/obj/item/disk/nuclear) in cast_on.get_contents())
+		owner.balloon_alert(owner, "may not enter refuge with the nuke disk!")
+		return FALSE
```

Comment:
I dislike this hardcoding because it doesn't work for similar items + can more than likely be worked around with shenanigans, 

You should be able to apply `TRAIT_SECLUDED_LOCATION` in the status effect, which (should) make all stationloving items in the contents of the mob move themselves out

Reply:
This is also a really long balloon alert